src/Components/ItemCard.js

```diff
@@ -1,28 +1,34 @@
-import  { Button, Card, Container} from "react-bootstrap";
+import { Button, Card, Container } from "react-bootstrap";
+import { useNavigate } from "react-router-dom";
 
 
-function ItemCard() {
+function ItemCard(props) {
+  
+  let navigate = useNavigate();
 
-    return (
-        <Container fluid style={{ display: 'flex', justifyContent: 'center' }}>
-
-            <div><Card style={{ width: '18rem' }}>
-                <Card.Img variant="top" src="isaac_asimov.jpeg" />
-                <Card.Body>
-                    <Card.Title><b>Isaac Asimov: The Complete Stories, Vol. 1</b></Card.Title>
-                    <Card.Text>
-                        Isaac Asimov best work
-                        In a world of robots how can we keep on keeping on
-                    </Card.Text>
-                    <Card.Footer>
-                        Price : $68.90
-                    </Card.Footer>
-                    <Button class="addToCartBtn" variant="primary">Add to cart</Button>
-                </Card.Body>
-            </Card> </div>
-        </Container>
-    
-    );
+  return (
+    <Container>
+      <Card style={{ width: "18rem", margin: "10%" }}>
+        <Card.Img variant="top" src={props.img} />
+        <Card.Body>
+          <Card.Title
+            onClick={() => {
```

Comment:
While the card title is clickable with the `onClick` function, we should add **an indication that allows the user to access an item's details page.** The image below shows that the element is clickable, but there is no indication that the element is clickable (regular cursor instead of link select cursor)

![image](https://user-images.githubusercontent.com/78051064/154867176-d1af72fe-d670-45c9-a978-bba80feca916.png)

Reply:
Yes, I was thinking of having a the title underlined when the cursor hovers the title. 

However, we are a bit short on time, so I thought it would be in another iteration.

Reply:
We should consider having the image clickable in Sprint 3. As part of the user experience, usability is an important component. The design must be simple, clean and intuitive. Users frequently click on images to access item description (intuitive).